doc/nrf/device_guides/working_with_nrf/nrf70/developing/raw_tx_operation.rst

```diff
@@ -0,0 +1,128 @@
+.. _ug_nrf70_developing_raw_tx_operation:
+
+Raw packet transmission
```

Comment:
Maybe change to "Sending Raw IEEE 802.11 packets", which includes a bit of more info.

Reply:
okay

Reply:
okay

Reply:
modified to Raw 802.11 packet transmission as heading